includes/Admin.php

```diff
@@ -82,4 +84,87 @@ public static function admin_body_class( $class ) {
 
 		return $class;
 	}
+
+	/**
+	 * Pre-fills post content with a web-story/embed block.
+	 *
+	 * @param string $content Default post content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	public static function prefill_post_content( $content ) {
+		$embed_story = isset( $_GET['from-web-story'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		if ( ! $embed_story ) {
+			return $content;
+		}
+
+		$post_id = absint( $_GET['from-web-story'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
I really don't like doing this. Why not escape correctly.

Reply:
Refactored this now